app/src/main/java/com/goldenraven/padawanwallet/ui/wallet/SendScreen.kt

```diff
@@ -281,15 +280,55 @@ fun Dialog(
     }
 }
 
-private fun broadcastTransaction(recipientAddress: String, amount: ULong, feeRate: Float = 1F) {
+fun verifyInput(
+    address: String,
+    amount: String,
+    feeRate: String,
+    snackbarHostState: SnackbarHostState,
+    scope: CoroutineScope,
+): Boolean {
+    if (address == "") {
```

Comment:
I suggest we use `isBlank()` instead, so as to make it more idiomatic and cleaner to read (as an added bonus this would also check for strings that are solely built of whitespace).

```kotlin
if (address.isBlank())
```